### PR TITLE
Final-touch

### PR DESCRIPTION
**Pull request by malikhaiderkhan**
In this part of project I implemented some css styling to look it better.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -4,9 +4,21 @@
   font-family: Arial, Helvetica, sans-serif;
 }
 
+body {
+  background-image: url('./images/lead5.png');
```
Comment (franclobo) [inline src/style.css:8]: - [ ] Kindly review your `webpack.config.js` and add the images loader to load the images successfully on your page, you can review this [article](https://webpack.js.org/guides/asset-management/#loading-images), besides correcting the styles of your page to match with the wireframe 👇🏽 

| Wireframe | Your design |
|---|---|
| image | image | 

- [ ] Finally, I suggest creating your own game to have a shorter score list 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @malikhaiderkhan, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Use webpack ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (malikhaiderkhan) [reply src/style.css:8]: @franclobo for creating own game I have to create new ID?

**Comment by malikhaiderkhan** [comment]
- Add the image loader successfully in webpack.config.js as suggested by reviewer.
- and the other part reviewer suggest to create own game I didn't understand the point because it's also not in the project requirements and I created my own ID '2ag8QvxcyAjEtOo0qNyP' from API by following the documentation provided by Microverse. 
https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/2ag8QvxcyAjEtOo0qNyP/scores/

So now I think that the project is now fully implemented according to the requirements.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @malikhaiderkhan ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Beautiful UI ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**